src/compositions/templates/case-study/Success/styled.js

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+import { textLighter, textLight } from 'styles/colours';
+
+export const Title = styled.h2`
+  font-size: 36px;
```

Comment:
Let's use rem for the font sizing. This is a multiplier of the root font size of the page which is defined in our global styles as 16px.

24 / 16 = 2.25 so we can set font-size: 2.25rem;